betka.sh

```diff
@@ -45,27 +43,23 @@ function load_configuration() {
 function check_os_and_parameters() {
   if [[ x"${OS}" == "xfedora" ]]; then
     if [[ x"${DOWNSTREAM_BRANCH}" == "x" ]]; then
-      DOWNSTREAM_BRANCH="main"
+      DOWNSTREAM_BRANCH="default"
```

Comment:
Ah I see, the `DOWNSTREAM_BRANCH` variable is now set to `default`. this should probably get renamed to something like `CWT_CONFIG` then

Reply:
How about making the `DOWNSTREAM_BRANCH` variable required as it was in the previous version and based on its content decide which config to use?
Eg. DOWNSTREAM_BRANCH=f34 -> f34.yaml, DOWNSTREAM_BRANCH=rhel-8.3.0 -> rhel8.yaml:rhel8.3

Reply:
I agree with renaming to CWT_CONFIG and for Fedora's case, we can use your proposal.
For the RHEL world, the user should know what he is doing and only check if the configuration is valid, like
`rhel8.yaml:rhel8.3` double dot is present and `rhel7` or `rhel8`.yaml file exists.

@pkubatrh Does it make sense to you?

Reply:
>For the RHEL world, the user should know what he is doing and only check if the configuration is valid, like
rhel8.yaml:rhel8.3 double dot is present and rhel7 or rhel8.yaml file exists.

I would expect the user to be a packager and know what branch they want to sync with. I would not necessarily expect the user to know what cwt config set to use. If we could transform the branch name to config name, it would be better from the user experience point of view.

Reply:
The function `convert_branch_to_cwt_tool` should do the magic